### PR TITLE
[FLOW-73] add Category

### PR DESCRIPTION
**Pull request by rmustafin**
**Intent (What)**

Add Category

https://developer.xero.com/documentation/practice-manager/categories

**Motivation (Why)**

We want `XpmRuby::Category` for our new cool features

**Implementation (How)**

**Consequence**

### CODE REVIEW

File: lib/xpm_ruby/category.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require "ox"
+
+module XpmRuby
+  module Category
+    extend self
+
+    class Error < Error; end
```
Comment (rudylee) [inline lib/xpm_ruby/category.rb:7]:
```suggestion
    class Error < StandardError; end
```

Comment (rudylee) [reply lib/xpm_ruby/category.rb:7]: Actually why do we need to define this here ? I noticed other modules also have this.

Comment (rmustafin) [reply lib/xpm_ruby/category.rb:7]: yeah, I think it might be a severe case of 🍝

Comment (rudylee) [reply lib/xpm_ruby/category.rb:7]: @abreckner might be able to answer. I'll say just delete it if we don't use it.

**Review by rudylee — APPROVED** [review]
Looks good, just one question about the error class

**Pull request merged**